src/app/pages/checkout-payment/checkout-payment/checkout-payment.component.ts

```diff
@@ -86,6 +94,11 @@ export class CheckoutPaymentComponent implements OnInit, OnChanges, OnDestroy {
       const redirect = params.get('redirect');
       this.redirectStatus = redirect;
     });
+
+    this.checkoutFacade.basketValidationResults$.pipe(takeUntil(this.destroy$)).subscribe(validationResults => {
+      // tslint:disable-next-line: no-boolean-literal-compare
+      this.basketHasError = validationResults.valid === false;
```

Comment:
You can probably find a clever solution here that doesn't need to disable a tslint rule. If you're concerned about differentiating between nullish values and false, I recommend checking out the [nullish coalescing operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing).